src/test/scala/nl/vroste/zio/kinesis/client/DynamicConsumerTest.scala

```diff
@@ -23,7 +23,9 @@ object DynamicConsumerTest extends DefaultRunnableSpec {
     (Console.live ++ Clock.live) >>> Logging.console() >>> Logging.withRootLoggerName(getClass.getName)
 
   private val env: ZLayer[Any, Throwable, Client with AdminClient with DynamicConsumer with Clock] =
-    (LocalStackServices.localHttpClient >>> LocalStackServices.kinesisAsyncClientLayer >>> (Client.live ++ AdminClient.live ++ (loggingLayer ++ LocalStackServices.localStackAwsLayer >>> DynamicConsumer.live))) ++ Clock.live
+    (LocalStackServices
```

Comment:
Looks like we're using localStackAwsLayer twice here.

Reply:
yes that was the case before